src/file_format.rst

```diff
@@ -0,0 +1,189 @@
+
+The first part of the file is called `the header`. The first line of the file
+is always a comment that LAMMPS does not read but which can contain
+information on the way the file was generated. Here we see the LAMMPS version
+and some more information like timestep and units. The following lines contain
+the number of `atoms` (125), the number of `atom types` (1) and three lines
+containing `xlo xhi` like indications. This header is simple, but generally,
+headers can contain much more information. The first two lines are explicit,
+you define a system with 125 atoms all of which have the same characteristics.
+The last three lines define the volume in which these atoms are contained. It
+is a box with edge coordinates starting at 0 and ending at 5 in every direction
+(x, y and z).
+
+From here starts the body of the file. The order of the sections is not important
+but all of them must come in the format:
+
+.. code-block:: LAMMPS
+
+   Section name # Capitalized, correctly spelled
+   # blank line
+   Section input # Number of line and format depend on the section.
+
+The first section you should see is the `Masses` section. In LAMMPS, masses are
+assigned to atom types so you only have one line here. All types have their own
+masses but several types can have the same mass. Types are LAMMPS way to refer
+to properties of particles that are *the same* for all particles of *the same
+type*. They are also used to determine how particles interact with one another.
+Types are not always binded to chemical species and you will see in further
+tutorials how it can be convenient to define different types for similar atoms.
+
+The `Atoms` section contains 125 lines, one per atom. The number on each line
+are ordered and are for each particle:
+* The particle ID which LAMMPS uses to refer to that particle internally
+* The type of the particle
+* The x, y and z coordinates in absolute distance value
+* The xflag, yflag and zflag values, which you can put aside for now
+
+Below you can also see the `Velocities` section which also contains 125 lines.
+Each lines gives the particle ID followed by the instantaneous velocities of
+the particles along the x, y and z axis in that order. The particle ID refers
+to the same ID as in the `Atoms` section.
+
+The format and meaning of the number in the `Atoms` and `Velocities` sections
+will depend on the `atom_style` the code is told to use. In most recents
+versions of LAMMPS, writing a data file with `write_data` will append a comment
+next to the `Atoms` section name (here `# atomic`). This helps human readers to
+know the meaning of the different number. Some formats can include more
+information like a molecule tag, particles' charges, particles' spins or
+orientation etc. For the velocities, some `atom_style` can require keeping
+track of angular momentum, angular velocities etc. You'll find a detailed
+description of each format in the `read_data section of the manual`_.
```

Comment:
```suggestion
This data file is quite simple because our Very First LAMMPS Script had the
`atom_style` `atomic`, where each particle only has an ID number 
and a three-dimensional position and velocity (and periodic image location).
More detailed simulations will use other `atom_style` commands,
such as `atom_style molecular`, which also tracks molecule ID
numbers and intramolecular arrangements like bonds,
angles, "dihedrals" and "impropers" (which you will learn about later).
You can see that LAMMPS includes a comment `# atomic` next to the
`Atoms` section name (here `# atomic`), and you can find a detailed
description of each format in the `read_data section of the manual`.
```

We regularly instruct our forum readers to use the latest LAMMPS version and we should assume they are doing the same when reading this manual (especially if they are using LAMMPS GUI).

Reply:
This is a very good remark. I will also add it in the introduction of the tutorial where I also tell readers to get in touch with their IT support to get LAMMPS installed. I think it might be necessary to mention the latest release is preferable.